tabs/system-setup/system-cleanup.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh -e
+
+. ../common-script.sh
+
+cleanup_system() {
+    printf "%b\n" "${YELLOW}Performing system cleanup...${RC}"
+    case $PACKAGER in
+        apt-get|nala)
+            $ESCALATION_TOOL "$PACKAGER" clean
```

Comment:
```suggestion
            "$ESCALATION_TOOL" "$PACKAGER" clean
```
Quote $escalation_tool for every one